asmcomp/s390x/emit.mlp

```diff
@@ -238,10 +238,11 @@ let int_literals = ref ([] : (nativeint * int) list)
 
 (* Masks for conditional branches after comparisons *)
 
+(* bit 0 = eq, bit 1 = lt, bit 2 = gt, bit 3 = overflow*)
 let branch_for_comparison = function
```

Comment:
You use "bit 0" to refer to the bit with weight 2^3...  But that's consistent with IBM's big-endian habits.